lib/admin/actions/organizations.js

```diff
@@ -75,7 +75,7 @@ export function deleteOrganization (organization) {
   return function (dispatch, getState) {
     dispatch(deletingOrganization())
     const url = `/api/manager/secure/organization/${organization.id}`
-    return secureFetch(url, getState(), 'delete')
+    return dispatch(secureFetch(url, 'delete'))
       .then(response => response.json())
```

Comment:
It seems like almost all of these secureFetches parse the json, perhaps the json can be parsed within secureFetch.